README.md

```diff
@@ -1,3 +1,241 @@
+* Use Conversation Slots to handle various inputs within one Node.
+
+![Flow](doc/source/images/architecture.png)
+
+### With Watson
+
+Want to take your Watson app to the next level? Looking to leverage Watson Brand assets? Join the [With Watson](https://www.ibm.com/watson/with-watson) program which provides exclusive brand, marketing, and tech resources to amplify and accelerate your Watson embedded commercial solution.
+
+## Included Components
+
+* [IBM Watson Conversation](https://www.ibm.com/watson/developercloud/conversation.html): Build, test and deploy a bot or virtual agent across mobile devices, messaging platforms, or even on a physical robot.
+
+## Featured technologies
+* [Node.js](https://nodejs.org/): An asynchronous event driven JavaScript runtime, designed to build scalable applications.
+
+# Watch the Video
+
+Coming Soon!
+
+# Steps
+
+**NOTE:** Perform steps 1-5 **OR** click the ``Deploy to Bluemix`` button and hit ``Create`` and then jump to step 6.
+
+
+1. [Clone the repo](#1-clone-the-repo)
+2. [Create Bluemix services](#2-create-bluemix-services)
+3. [Get Bluemix credentials and add to .env](#3-get-creds-add-to-.env)
+4. [Configure Watson Conversation](#4-configure-watson-conversation)
+5. [Run the application](#5-run-the-application)
+
+## 1. Clone the repo
+
+Clone `watson-conversation-slots-intro` locally. In a terminal, run:
+
+  `$ git clone https://github.com/ibm/watson-conversation-slots-intro`
+
+We’ll be using the file [`data/pizza-advanced.json`](data/pizza-advanced.json) to upload
+the Conversation Intents, Entities, and Dialog Nodes.
+
+## 2. Create Bluemix services
+
+Create the following service:
+
+  * [**Watson Conversation**](https://console.ng.bluemix.net/catalog/services/conversation)
+
+## 3. Get Bluemix Services Credentials and add to .env file
+
+As you create the Blumix Services, you'll need to create service credentials and get the
+username and password:
+
+
+<p align="center">
+  <img width="500" height="350" src="doc/source/images/WatsonCred1.png">
+</p>
+
+Move the watson-conversation-slots-intro/env.sample file to /.env and populate the service
+credentials as you create the credentials:
+
+```
+# Watson conversation
+CONVERSATION_USERNAME=<add_conversation_username>
+CONVERSATION_PASSWORD=<add_conversation_password>
+WORKSPACE_ID=<add_conversation_workspace>
+```
+
+## 4. Configure Watson Conversation
+
+Launch the **Watson Conversation** tool. Use the **import** icon button on the right
+
+<p align="center">
+  <img width="400" height="55" src="doc/source/images/import_conversation_workspace.png">
+</p>
+
+Find the local version of [`data/pizza-advanced.json`](data/pizza-advanced.json) and select
+**Import**. Find the **Workspace ID** by clicking on the context menu of the new
+workspace and select **View details**.
+
+<p align="center">
+  <img width="400" height="250" src="doc/source/images/open_conversation_menu.png">
+</p>
+
+ Put this Workspace ID into the .env file
+as WORKSPACE_ID.
```

Comment:
Done..if you mean but a backtick on either side of WORKSPACE_ID.
I think that comment was rendered in some weird way..